api/metrics-api-common/src/main/java/org/hawkular/metrics/api/jaxrs/util/MetricTypeTextConverter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2014-2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.metrics.api.jaxrs.util;
+
+import org.hawkular.metrics.core.api.MetricType;
+
+/**
+ * Transforms from longForm of the metrics (such as gauges) used by the REST-interface to internal
```

Comment:
Yea, they're variable names I've used on client side to separate the naming strategies (I still think we should get rid of these or accept both..)